### PR TITLE
Update default annotations and documentation for web server service

### PR DESCRIPTION
**Pull request by anr-tonic**
Updates EKS annotations to include the replacement for `aws-load-balancer-internal: true` 

https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.5/guide/service/annotations/#lb-scheme

Provides updated documentation on the load balancer creation as default plus links to documentation for AWS, Azure and GCP on annotations that could be applied if specific behavior is needed.

### CODE REVIEW

**Review by calexander3 — DISMISSED** [review]

**Review by daniel-tonic — CHANGES_REQUESTED** [review]
I think we should hold off on this for a bit.  On Azure, I think this will result in Tonic being exposed on a public IP by default.  I would like to test what this actually does before we merge in this change.

Also, this would mean that TIM would not longer be usable to install Tonic on cloud managed k8s.

**Comment by calexander3** [comment]
We need to at least do the aws stuff right now since it is definitely creating a public IP and its holding up a customer from being able to use TIM

**Comment by daniel-tonic** [comment]
I don't think this will fix the public IP thing on EKS.  I think the public IP thing on EKS might be caused by our use of a deprecated annotation.  After all, the annotation is called "load-balancer-internal".  If I'm right, EKS is actually ignoring this which is why we're getting a public IP so this change won't solve the problem.

**Review by calexander3 — APPROVED** [review]

**Review by daniel-tonic — APPROVED** [review]

**Pull request merged**
